### PR TITLE
Update extensions.xml

### PR DESCRIPTION
**Pull request by azakhary**
Version 0.1.1 supporting latest editor and latest libGDX released

### CODE REVIEW

File: extensions/gdx-setup/src/com/badlogic/gdx/setup/data/extensions.xml [hunk 1]
```diff
@@ -2,10 +2,10 @@
 <extensions>
 	<extension>
 		<name>Overlap2D</name>
-		<description>Level and UI Editor Runtime</description>
+		<description>Runtime for Overlap2D ui and level editing toolkit</description>
```
Comment (saltares) [inline extensions/gdx-setup/src/com/badlogic/gdx/setup/data/extensions.xml]: Lowercase for UI? Shouldn't it be uppercase?

File: extensions/gdx-setup/src/com/badlogic/gdx/setup/data/extensions.xml [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 		<package>com.underwaterapps.overlap2druntime</package>
-		<version>0.1.0</version>
-		<compatibility>1.6.2</compatibility>
+		<version>0.1.1</version>
+		<compatibility>1.6.4</compatibility>
```
Comment (saltares) [inline extensions/gdx-setup/src/com/badlogic/gdx/setup/data/extensions.xml]: Is it not compatible with 1.6.5?

**Comment by azakhary** [comment]
Oh 1.6.5 is up, forgot that. Well, I'll make it compatible and re-push. stand by :)

**Comment by azakhary** [comment]
I think it should be compatible with 1.6.5 because nothing almost changed from previous version. but it says in it's dependencies 1.6.4. Will this work ?

**Comment by azakhary** [comment]
yep, checked it works fine with 1.6.5. But it says in it's gradle dependency 1.6.4 and I can't change that without building new version to maven. So this should do right?

**Comment by azakhary** [comment]
Should I put 1.6.5. here in the extension config?

**Comment by saltares** [comment]
If there are no API breaking changes for you, go ahead.

**Comment by azakhary** [comment]
Done.

**Comment by azakhary** [comment]
I checked, all works, no API changes that can affect anything

**Comment by saltares** [comment]
Can you squash the commits please?

**Comment by azakhary** [comment]
I can, but in this particular case it's so much easier to do new PR if thats ok?

**Pull request closed without merging**